src/Compilers/Test/Utilities/CSharp/TestOptions.cs

```diff
@@ -25,7 +25,7 @@ public static class TestOptions
         public static readonly CSharpParseOptions RegularWithDocumentationComments = Regular.WithDocumentationMode(DocumentationMode.Diagnose);
         public static readonly CSharpParseOptions RegularWithLegacyStrongName = Regular.WithFeature("UseLegacyStrongNameProvider");
         public static readonly CSharpParseOptions WithoutImprovedOverloadCandidates = Regular.WithLanguageVersion(MessageID.IDS_FeatureImprovedOverloadCandidates.RequiredVersion() - 1);
-
+        public static readonly CSharpParseOptions RegularWithPreview = Regular.WithLanguageVersion(LanguageVersion.Preview);
```

Comment:
I hate to point this out but this bears a striking resemblance to `TestOptions.RegularPreview`.